views/templates/block/checklist.twig

```diff
@@ -133,30 +129,84 @@
                         {% endif %}
                     </td>
                 </tr>
+                {% if not checkPhpVersionCompatibility %}
+                  <tr>
+                    <td>{{ 'Your current PHP version isn\'t compatible with your PrestaShop version.'|trans }}</td>
+                    <td>{{ icons.nok }}</td>
+                  </tr>
+                {% endif %}
+                {% if not checkApacheModRewrite %}
+                  <tr>
+                    <td>{{ 'Apache mod_rewrite is disabled.'|trans }}</td>
+                    <td>{{ icons.nok }}</td>
+                  </tr>
+                {% endif %}
+                {% if notLoadedPhpExtensions|length > 0 %}
+                  <tr>
+                    <td>
+                      {{ 'The PHP extension are not installed: %s.'|trans([notLoadedPhpExtensions|join(', ')]) }}
```

Comment:
The PHP extension pdo_mysql, intl, curl, json, simplexml, zip, fileinfo, mbstring, opensssl_encrypt is/are not installed (display only the one(s) that are not installed).

From what I understand of the issue, there should be several strings (one for singular, one for plural). 

'The following PHP extension is not installed: %s.' and 'The following PHP extensions are not installed: %s.'

Reply:
Fixed